fourier/fourier_test.go

```diff
@@ -0,0 +1,234 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
```

Comment:
It would be nice to have one bigger input that we compare with numpy, like a square wave example or something.

Reply:
OK